src/parsec.rs

```diff
@@ -293,6 +294,17 @@ impl ParsecMap {
     }
 }
 
+// Generate a DkgResult that can be used for the first node
+pub fn generate_first_dkg_result(rng: &mut MainRng) -> DkgResult {
+    let threshold = 0;
+    let index = 0;
+    let rng = rng::new_from(rng);
```

Comment:
It's not necessary to create new rng here. Just use the one passed to this function.